.travis.yml

```diff
@@ -23,7 +23,8 @@ branches:
 addons:
   apt:
     sources:
-    - sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main'
+# TODO: change links to bionic when packages there will not crash travis
```

Comment:
This TODO can be removed now, and the commit message edited as you are using the bionic packages again.